src/pages/_app.js

```diff
@@ -1,3 +1,4 @@
+import "docsearch.js/dist/cdn/docsearch.min.css";
```

Comment:
I dont think you need that. We are adding that in the head already, but if that works, perhaps you can remove inserting css in the seo.js component.